README.md

```diff
@@ -1 +1,83 @@
-# OOP-mini-projects-set-up-associations
\ No newline at end of file
+# school_library_associations_training
+
+## About
+
+This is a code review training project for Ruby/Rails.
+
+There are 7 classes and these classes are connected through database associations.
+
+## Built with
+* Postgresql
+* rubocop linters
+- <code><img height="20" src="https://www.ruby-lang.org/images/header-ruby-logo.png"></code><br>
+-  <code><img height="20" src="https://rubyonrails.org/images/rails-logo.svg"></code><br>
+- ![Git](https://img.shields.io/badge/-Git-05122A?style=flat&logo=git)&nbsp;<br>
+- ![GitHub](https://img.shields.io/badge/-GitHub-05122A?style=flat&logo=github)&nbsp;<br>
+- ![Visual Studio Code](https://img.shields.io/badge/-Visual%20Studio%20Code-05122A?style=flat&logo=visual-studio-code&logoColor=007ACC)&nbsp;
```

Comment:
- [ ] <b>REQUIRED:</b> There is no point of listing `PostgreSQL` and `Rails` Here, because we are not using them in this project. Kindly remove them.